content/source/layouts/runscope.erb

```diff
@@ -0,0 +1 @@
+../../../ext/providers/PROVIDER/website/runscope.erb
```

Comment:
This symlink still has `PROVIDER` in its path — gotta replace it with `runscope` for this to work.